spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js

```diff
@@ -16,6 +16,9 @@ import { WiredScreenshareChangeListener } from '../../wired-screenshare';
 import AbstractMeetingFrame from '../AbstractMeetingFrame';
 import MeetingType from '../MeetingType';
 import FeedbackHider from './FeedbackHider';
+import { getClientAspectRatio,
```

Comment:
Sort this import correctly, deeper imports go above, after builtins. Didn't the linter catch it?

Also, when importing multiple items in multiple lines, keep each item in a new line.

Reply:
I think I have a problem with the eslint ;( "Error: Failed to load plugin 'flowtype' declared in 'spot-client/.eslintrc » eslint-config-jitsi': Cannot find module 'eslint-plugin-flowtype'"